lib/create_game.rb

```diff
@@ -0,0 +1,20 @@
+class Player
+  attr_reader :name
+
+  def initialize(name)
+    @name = name
+  end
+end
+
+class Newboard
```

Comment:
As per the Ruby convention, class names should follow the PascalCase naming convention instead of capitalized words put together. In this case, the name of the class should be `NewBoard`.

- [ ] Please rename this class to follow PascalCase.